lib/resources/interface.rb

```diff
@@ -41,6 +43,30 @@ def speed
       interface_info.nil? ? nil : interface_info[:speed]
     end
 
+    def ipv4_addressable?
+      !ipv4_addresses.nil? && !ipv4_addresses.empty?
+    end
+
+    def ipv6_addressable?
+      !ipv6_addresses.nil? && !ipv6_addresses.empty?
+    end
+
+    def ipv4_addresses
+      ipv4_cidrs.map { |i| i.split('/')[0] }
+    end
+
+    def ipv6_addresses
+      ipv6_cidrs.map { |i| i.split('/')[0] }
+    end
+
+    def ipv4_cidrs
```

Comment:
While I think this is useful, I think some people might want to see it as an netmask, too.  That could be a separate feature request if you feel that isn't worth it.

Reply:
I can give it a go.